api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/helpers/HeadcountHelper.java

```diff
@@ -48,11 +44,7 @@ public void setComparisonValues(List<HeadcountHeader> headcountHeaderList, List<
   }
 
   public UUID getPreviousSeptemberCollectionID(SdcSchoolCollectionEntity sdcSchoolCollectionEntity) {
-    int previousYear = sdcSchoolCollectionEntity.getCreateDate().minusYears(1).getYear();
-    LocalDateTime startOfCollectionDate = LocalDate.of(previousYear, Month.SEPTEMBER, 1).atTime(LocalTime.MIN);
-    LocalDateTime endOfCollectionDate = LocalDate.of(previousYear, Month.SEPTEMBER, 30).atTime(LocalTime.MAX);
-
-    var septemberCollection = sdcSchoolCollectionRepository.findAllBySchoolIDAndCreateDateBetween(sdcSchoolCollectionEntity.getSchoolID(), startOfCollectionDate, endOfCollectionDate);
+    var septemberCollection = sdcSchoolCollectionRepository.findAllCollectionsForSchoolInLastTwoYears(sdcSchoolCollectionEntity.getSchoolID(), sdcSchoolCollectionEntity.getSdcSchoolCollectionID());
     if(!septemberCollection.isEmpty()) {
```

Comment:
I don't think this will work with the new query.  This is to for the 'compare to previous september collection' button and was used to get the previous september collection

Reply:
I made significant changes to the query to pull the previous September collection. Please take a look.